src/mavoscript.js

```diff
@@ -321,7 +321,62 @@ var _ = Mavo.Script = {
 				return ret;
 			},
 			precedence: 3
-		}
+		},
+		"groupby": {
+			symbol: "by",
+			code: (array, key) => {
+				if (Array.isArray(array) && Array.isArray(key))  { 
+					var propName = key[Mavo.as] === true ? key[Mavo.name] : $.value(key[0], Mavo.toNode, "property");
```

Comment:
You don't need to do `=== true`, just checking `key[Mavo.as]` suffices.
Same for all the other Symbol properties.

Also, this is super minor, but can we call this variable `property` or `name`? (I don't think we have another one with that name in the same scope?). I generally try to use familiar variable names across scopes, as it gzips better (and aids readability if e.g. a property name is always called `property`). Also, there is recent research that abbreviations hurt code comprehension, so I avoid those too.